app/styles/_integrations.less

```diff
@@ -0,0 +1,43 @@
+.service-integration {
+  width: 33%;
```

Comment:
bootstrap has grid system you can use -> https://getbootstrap.com/docs/3.3/css/#grid

Reply:
when I tried using col-md-4 for this, it pops it out of the layout, like this:
<img width="637" alt="screen shot 2018-03-02 at 11 48 04" src="https://user-images.githubusercontent.com/1033452/36897720-9bcc2e3e-1e0f-11e8-88f0-628300d7175a.png">
Any suggestions on which I should use?